### PR TITLE
fix/add linting, include siren-entity.html as a component

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "d2l-sequence-navigator",
```
Comment (jopnick) [inline package.json]: This doesn't seem like it belongs here

File: siren-entity.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<link rel="import" href="../../polymer/polymer-element.html">
```
Comment (jopnick) [inline siren-entity.html]: This isn't really a mixin so much as a component which consumes the mixins. perhaps we can move it to a utility repo in `BrightspaceHypermediaComponents` org

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,4 @@ bower_components/
 build/
 node_modules/
 npm-debug.log
+.idea/
```
Comment (jopnick) [inline .gitignore]: whats this one do, what uses `.idea`?

Comment (jwong-d2l) [reply .gitignore]: webstorm files

File: package.json (hunk 1 above)
Comment (jwong-d2l) [reply package.json]: fixed

**Comment by jwong-d2l** [comment]
removed siren-entity.  Where did you end up putting it?

**Comment by jopnick** [comment]
This looks just super, entity can be found 

https://github.com/BrightspaceHypermediaComponents/siren-entity

**Pull request merged**